src/protocols/bdx/TransferFacilitator.h

```diff
@@ -79,12 +84,18 @@ class TransferFacilitator : public Messaging::ExchangeDelegate, public Messaging
      * The callback for when the poll timer expires. The poll timer regulates how often the TransferSession is polled.
      */
     static void PollTimerHandler(chip::System::Layer * systemLayer, void * appState);
+    static void ImmediatePollTimerHandler(chip::System::Layer * systemLayer, void * appState);
```

Comment:
The difference between these two handlers needs to be clearly documented.  It's not obvious from the name.